Server/User/dependencies.py

```
@@ -41,15 +41,38 @@ async def get_current_user(
+            detail="Your account is suspended"
+        )
+    if user.status == "inactive":
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="Your account is inactive"
+        )
+    
     return user
 
 
 def get_current_active_user(current_user: User = Depends(get_current_user)) -> User:
```

Ruff (0.14.4)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)